src/rgw/rgw_lc.cc

```diff
@@ -147,10 +147,9 @@ void *RGWLC::LCWorker::entry() {
 
     utime_t end = ceph_clock_now();
     int secs = schedule_next_start_time(start, end);
-    time_t next_time = end + secs;
-    char buf[30];
-    char *nt = ctime_r(&next_time, buf);
-    dout(5) << "schedule life cycle next start time: " << nt <<dendl;
+    end += secs;
```

Comment:
This might be a bit misleading, I think. We've already finished the turn in time `end` but, in this line, we're reusing the variable for something different. Maybe just:

```cpp
    dout(5) << "schedule life cycle next start time: "
            << rgw_to_asctime(end + secs) <<dendl;
```